controllers/storagecluster/topology.go

```diff
@@ -96,13 +96,11 @@ func determinePlacementRack(
 	rackList := []string{}
 
 	if len(nodeRacks.Labels) < minRacks {
-		for i := len(nodeRacks.Labels); i < minRacks; i++ {
-			for j := 0; j <= i; j++ {
```

Comment:
I really don't remember why this was here... I'm pretty sure I wrote it. Maybe it was an artifact from an abandoned implementation I never removed. Ah well, happy to see it gone. :)